components/org.wso2.carbon.identity.sts.mgt/src/main/java/org/wso2/carbon/identity/sts/mgt/base/IdentityBaseUtil.java

```diff
@@ -0,0 +1,151 @@
+/*
+ *  Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Lets change the license header.

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-sts/pull/94/commits/5504ab5c7e661e2e7467808b0adfdba1cb3c154a.